features/rhelah/atomic/atomic_scanner.feature

```diff
@@ -90,6 +90,68 @@ Background: Atomic hosts are discovered
   Scenario: 16. Remove all of containers
        When docker remove all of containers
 
+  @pull_rsyslog_image
+  Scenario: 17. Pull latest rhel7/rsyslog image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/rsyslog" from repository
+       Then Check whether "rsyslog" is installed
+
+  @pull_sadc_image
+  Scenario: 18. Pull latest rhel7/sadc image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/sadc" from repository
+       Then Check whether "sadc" is installed
+
+  @pull_sssd_image
+  Scenario: 19. Pull latest rhel7/sssd image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/sssd" from repository
+       Then Check whether "sssd" is installed
+
+  @pull_etcd_image
+  Scenario: 20. Pull latest rhel7/etcd image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/etcd" from repository
+       Then Check whether "etcd" is installed
+
+  @pull_kubernetes_apiserver_image
+  Scenario: 21. Pull latest rhel7/kubernetes-apiserver image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/kubernetes-apiserver" from repository
+       Then Check whether "kubernetes-apiserver" is installed
+
+  @pull_kubernetes_scheduler_image
+  Scenario: 22. Pull latest rhel7/kubernetes-scheduler image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/kubernetes-scheduler" from repository
+       Then Check whether "kubernetes-scheduler" is installed
+
+  @pull_kubernetes_controller_mgr_image
+  Scenario: 23. Pull latest rhel7/kubernetes-controller-mgr image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/kubernetes-controller-mgr" from repository
+       Then Check whether "kubernetes-controller-mgr" is installed
+
+  @pull_pod_infrastructure_image
+  Scenario: 24. Pull latest rhel7/pod-infrastructure image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/pod-infrastructure" from repository
+       Then Check whether "pod-infrastructure" is installed
+
+  @pull_cockpit_ws_image
+  Scenario: 25. Pull latest rhel7/cockpit-ws image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/cockpit-ws" from repository
+       Then Check whether "cockpit-ws" is installed
+
+  @pull_rhel6_image
+  Scenario: 26. Pull latest rhel6 image from repository
+       When atomic update latest "registry.access.redhat.com/rhel6" from repository
+       Then Check whether "rhel6" is installed
+
+  @pull_rhel_tools_image
+  Scenario: 27. Pull latest rhel7/rhel-tools image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/rhel-tools" from repository
+       Then Check whether "rhel-tools" is installed
```

Comment:
Could all these scenarios be condensed into a single scenario with an outline?

See  http://pythonhosted.org/behave/gherkin.html#scenario-outlines

```
Scenario: 17. Pull latest <image> image from repository
       When atomic update latest "registry.access.redhat.com/<image>" from repository
       Then Check whether "<image>" is installed

       Examples: Images
       | image            |
       | rhel7/rhel-tools |
       | rhel7/rsyslog    |
       | rhel7/sadc       |
```